lib/db/files_db.dart

```diff
@@ -968,6 +987,8 @@ class FilesDB {
     );
   }
 
+  // getLocalFiles is used in multiple places. Do not add dedupte logic
+  // while
```

Comment:
Incomplete comment? o_O
But as discussed you could just add a `shouldDedupe` param here and move on

Reply:
Will add another method here, also need to add check on the ownership of file.